app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,4 @@
 <resources>
-    <string name="app_name">KotLinExample</string>
+
+    <string name="title">Title</string>
```

Comment:
WARNING: [UnusedResources] The resource `R.string.title` appears to be unused
 Unused resources make applications larger and slow down builds.